demo/src/static/workers/flight-path-data-decoder.js

```diff
@@ -0,0 +1,55 @@
+importScripts('./util.js');
+const FLUSH_LIMIT = 20000;
+let result = [];
+let index = 0;
+let count = 0;
+
+onmessage = function(e) {
+  const lines = e.data.text.split('\n');
+
+  lines.forEach(function(line) {
+
+    if (!line) {
+      return;
+    }
+
+    var parts = line.split('\t');
+    var coords0 = parts[2].split('\x01').map(function(str) { return decodePolyline(str, 5) });
+    var coords1 = parts[3].split('\x01').map(function(str) { return decodePolyline(str, 1) });
+
+    coords0.forEach(function(lineStr, i) {
+      for (var j = 1; j < lineStr.length; j++) {
+        var prevPt0 = coords0[i][j - 1],
```

Comment:
> Workers are not compiled, so we can't use es6 syntax. 

Ok thanks for clarifying, I didn't realize that. I will need to study the setup then to understand the constraints. 
1) We could certainly just run them all through a babel step - at the cost of losing hot reloading.
```
"build-workers": "babel src/workers --out-dir dist-workers ... --source-maps"
```
2) It is probably a bit early to require ES6 compatible browsers, but essentially all the browsers we want to support from a WebGL perspective anyway (the evergreen browsers) already run untranspiled ES6.